### ISSUE TITLE
Crashes and undefined behavior when changing "no dead"

### ISSUE THREAD

**Issue by Sarroxxie** [veeenu/sekiro-practice-tool#2]
Today I used the practice tool for the first time. Everything worked great except the "no dead" option. I got consistent crashes when turning this option on via ImGUI or via hotkey. As the discord invite that is linked in the README is an expired link, I've tried my best to research the bug before opening this issue.

**Efforts I tried to make the bug go away:**
I'll just list the things I tried after getting consistent crashes (sadly all in vain):
- switching Sekiro to windowed mode
- switching Sekiro to offline mode
- switching Steam to offline mode
- adding and removing the "dinput8.dll" that got shipped with the tool
- redownloading the practice tool
- restarting PC
- updating graphics drivers to latest version via clean install
- completely reinstalling the game
- deleting the AppData content of Sekiro

**Other anomalies I noticed:**
- sometimes the checkbox of "no dead" in the ImGUI was immediately activated when starting the tool
- sometimes the checkbox of "no dead" in the ImGUI would automatically activate when I died by jumping off the map
- once the blood particles were rendered at the wrong location (somewhere in the top left of the screen) and only as multiple red circles when "no dead" was activated

I've tested all the other features of the practice tool and they worked like a charm. If you need additional information or testing from my side, feel free to hit me up. As far as I could tell, the log file looked normal that I produced after activating log_level "TRACE" and setting dxgi_debug to "true". I attached the compressed log file so you could see for yourself.

tl;dr: 99% of the times I toggled "no dead" I got crashes.

**Log File:**
[jdsd_sekiro_practice_tool_log.zip](https://github.com/veeenu/sekiro-practice-tool/files/11943786/jdsd_sekiro_practice_tool_log.zip)

**Comment by veeenu** [comment]
I got similar reports today, must've missed this notification. Everything points to the fact that the no dead flag must be wrong.

Thank you for reporting this, I will look into it!
